selfdrive/ui/translations/main_zh-CHT.ts

```diff
@@ -1004,19 +1004,19 @@ location set</source>
     </message>
     <message>
         <source>Experimental Mode</source>
-        <translation type="unfinished"></translation>
+        <translation>實驗模式</translation>
     </message>
     <message>
         <source>    openpilot defaults to driving in &lt;b&gt;chill mode&lt;/b&gt;.    Experimental mode enables &lt;b&gt;alpha-level features&lt;/b&gt; that aren&apos;t ready for chill mode.     Experimental features are listed below:    &lt;br&gt;     &lt;h4&gt;🌮 End-to-End Longitudinal Control 🌮&lt;/h4&gt;     Let the driving model control the gas and brakes. openpilot will drive as it thinks a human would, including stopping for red lights and stop signs. Since the driving model decides which speed to drive, the set speed will only act as an upper bound.</source>
-        <translation type="unfinished"></translation>
+        <translation>    openpilot 默認以 &lt;b&gt;輕鬆模式&lt;/b&gt; 駕駛。    實驗模式啟用了尚未準備好進入輕鬆模式的 &lt;b&gt;alpha 級功能&lt;/b&gt;。     實驗功能如下：    &lt;br&gt;     &lt;h4&gt;🌮端到端縱向控制🌮&lt;/h4&gt;     讓駕駛模型控制油門和剎車。 openpilot 將像人類一樣駕駛，包括紅燈和停車標誌時停車，因為是由駕駛模型來決定車速，所以定速的設定值只會作為上限。</translation>
```

Comment:
@eFiniLan this was a bit confusing, I'll try to clean it up. In Qt Linguist, the elevated dots are showing you spaces, not periods/ellipsis. Fixed!